<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Radio3DRConfig</class>
 <widget class="QWidget" name="Radio3DRConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;h2&gt;3DR Radio&lt;/h2&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="loadSettingsButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveSettingsButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Settings</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="localGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>321</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Local</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="3">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>LBT RSSI</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Max Freq.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QCheckBox" name="eccCheckBox">
      <property name="text">
       <string>ECC</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Net ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QComboBox" name="minFreqComboBox"/>
    </item>
    <item row="8" column="3">
     <widget class="QLabel" name="label_12">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Node Dest.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QComboBox" name="nodeDestComboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QComboBox" name="nodeCountComboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Format</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="label_11">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Node ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QCheckBox" name="opResendCheckBox">
      <property name="text">
       <string>Op Resend</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Min Freq.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Air Baud</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Tx Power</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string># Channels</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="formatLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="airBaudComboBox"/>
    </item>
    <item row="9" column="2" colspan="2">
     <widget class="QLabel" name="label_13">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Node Count</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="txPowerComboBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Baud</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QComboBox" name="maxFreqComboBox"/>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QTextEdit" name="rssiTextEdit">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.Lucida Grande UI'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Lucida Grande';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QComboBox" name="nodeIdComboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>RSSI</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="baudComboBox"/>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Duty Cycle</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="versionLabel">
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_1">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Version</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="netIdSpinBox">
      <property name="maximum">
       <number>499</number>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QSpinBox" name="dutyCycleSpinBox">
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QSpinBox" name="lbtRssiSpinBox">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QSpinBox" name="numChannelsSpinBox">
      <property name="maximum">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_32">
      <property name="text">
       <string>MAVLink</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QComboBox" name="mavLinkComboBox"/>
    </item>
    <item row="10" column="1">
     <widget class="QSpinBox" name="maxWindowSpinBox">
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>Max Win</string>
      </property>
     </widget>
    </item>
    <item row="10" column="4">
     <widget class="QComboBox" name="rtsCtsComboBox"/>
    </item>
    <item row="10" column="3">
     <widget class="QLabel" name="label_38">
      <property name="text">
       <string>HW Flow</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>label_1</zorder>
   <zorder>label_2</zorder>
   <zorder>formatLineEdit</zorder>
   <zorder>label_3</zorder>
   <zorder>label_6</zorder>
   <zorder>minFreqComboBox</zorder>
   <zorder>maxFreqComboBox</zorder>
   <zorder>baudComboBox</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_7</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>nodeIdComboBox</zorder>
   <zorder>label_12</zorder>
   <zorder>nodeDestComboBox</zorder>
   <zorder>label_13</zorder>
   <zorder>nodeCountComboBox</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>airBaudComboBox</zorder>
   <zorder>label_14</zorder>
   <zorder>label_17</zorder>
   <zorder>txPowerComboBox</zorder>
   <zorder>eccCheckBox</zorder>
   <zorder>opResendCheckBox</zorder>
   <zorder>rssiTextEdit</zorder>
   <zorder>versionLabel</zorder>
   <zorder>netIdSpinBox</zorder>
   <zorder>dutyCycleSpinBox</zorder>
   <zorder>lbtRssiSpinBox</zorder>
   <zorder>numChannelsSpinBox</zorder>
   <zorder>label_32</zorder>
   <zorder>mavLinkComboBox</zorder>
   <zorder>maxWindowSpinBox</zorder>
   <zorder>label_34</zorder>
   <zorder>rtsCtsComboBox</zorder>
   <zorder>label_38</zorder>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>651</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="remoteGroupBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>321</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Remote</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Net ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QComboBox" name="minFreqComboBox_remote"/>
    </item>
    <item row="9" column="1">
     <widget class="QCheckBox" name="opResendCheckBox_remote">
      <property name="text">
       <string>Op Resend</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Air Baud</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Min Freq.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Tx Power</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string># Channels</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="formatLineEdit_remote">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="airBaudComboBox_remote"/>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>LBT RSSI</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="txPowerComboBox_remote"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Baud</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QComboBox" name="maxFreqComboBox_remote"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_27">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Version</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>Duty Cycle</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="baudComboBox_remote"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>RSSI</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>Format</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QTextEdit" name="rssiTextEdit_remote"/>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Max Freq.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QCheckBox" name="eccCheckBox_remote">
      <property name="text">
       <string>ECC</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="versionLabel_remote">
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="netIdSpinBox_remote">
      <property name="maximum">
       <number>499</number>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QSpinBox" name="dutyCycleSpinBox_remote">
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QSpinBox" name="lbtRssiSpinBox_remote"/>
    </item>
    <item row="4" column="4">
     <widget class="QSpinBox" name="numChannelsSpinBox_remote">
      <property name="maximum">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QComboBox" name="nodeCountComboBox_remote">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="2" colspan="2">
     <widget class="QLabel" name="label_25">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Node Count</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QComboBox" name="nodeDestComboBox_remote">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QLabel" name="label_19">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Node Dest.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="label_20">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Node ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QComboBox" name="nodeIdComboBox_remote">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string>MAVLink</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QComboBox" name="mavLinkComboBox_remote"/>
    </item>
    <item row="10" column="1">
     <widget class="QSpinBox" name="maxWindowSpinBox_remote">
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_35">
      <property name="text">
       <string>Max Win</string>
      </property>
     </widget>
    </item>
    <item row="10" column="4">
     <widget class="QComboBox" name="rtsCtsComboBox_remote"/>
    </item>
    <item row="10" column="3">
     <widget class="QLabel" name="label_37">
      <property name="text">
       <string>HW Flow</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="copyToRemoteButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy Required
Items to Remote</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>66</y>
     <width>131</width>
     <height>137</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <item>
     <widget class="QLabel" name="label_31">
      <property name="text">
       <string>Link Settings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="linkPortComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>123</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="baudPortComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>123</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>91</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="settingsButton">
      <property name="text">
       <string>Adv. Settings</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="localStatus">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>440</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>STATUS:</string>
   </property>
  </widget>
  <widget class="QLabel" name="remoteStatus">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>440</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>STATUS:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetDefaultsButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore Defaults</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_36">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>230</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please set radio settings &lt;br/&gt;as per your countries laws&lt;br/&gt;see &lt;a href=&quot;http://copter.ardupilot.com/wiki/common-using-the-3dr-radio-for-telemetry-with-apm-and-px4/#Support_for_different_countriesregions&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;http://ardupilot.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
